.castor/docker.php

```diff
@@ -150,7 +150,7 @@ function destroy(
     fs()->remove($files);
 }
 
-#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)', namespace: '')]
+#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)', aliases: ['generate-certificates'])]
```

Comment:
Hmm, I don't think it's really useful. We run this comment only once (and actually never because the very first time on a projet, you run `castor start`)

It might conflict with other commands.

Could you revert this changes?